data/json/monsters/feral_humans.json

```diff
@@ -45,6 +45,7 @@
       }
     ],
     "death_drops": "feral_humans_death_drops_pipe",
+    "upgrades": { "half_life": 90, "into_group": "GROUP_ZOMBIE_UPGRADE" },
```

Comment:
Here is your problem.  You've chained copy from so that all your infected copy an `into_group` from here while also adding `into` in the first version of the `mon_civilian_panic`.  You'd need to extend delete this or change your into's to an into_group replacing the one copied from.

Reply:
Thanks for the hint. I tried but I am doing this blind since I can not ignore the `Monster attacks must specify type and/or id` error I get when I apply this PR locally. This iteration of blindly trying things has not worked, unfortunately.

Reply:
OK since the error didn't change that means you can't use delete to delete "into" that's good to learn.  The only thing you can do then is not copy-from a monster with `into` and make a monster that uses `into_group`.   Above are you saying if you don't use copy-from it breaks the PR locally?

Reply:
Okay, since I took a little break from working on this, I can now see that the error I could not ignore is no longer present in experimental. Hooray, I can continue testing!

I tried the following, which all do not work:

```json
"delete": { "upgrades": [] },
"delete": { "upgrades" },
"delete": { "upgrades": {} },
"delete": { "upgrades": { "into_group": "GROUP_ZOMBIE_UPGRADE", "half_life": 90 } },
```

Reply:
Irwiss suggested I use a `GROUP_NULL` override and it worked!